collector/test/AbortHandler_test.cpp

```diff
@@ -31,6 +31,11 @@ void test_crash() {
 }
 
 TEST(AbortHandler, Crash) {
+
+  #if defined(__powerpc64__)
+    GTEST_SKIP() << "Skipping AbortHandler test on ppc64le";
+  #endif
```

Comment:
There's a linting error here, running `make -C collector format` should solve it.